fri/src/prover/tests.rs

```diff
@@ -107,10 +115,58 @@ pub fn verify_proof(
     )
     .unwrap();
     let mut coin = RandomCoin::<BaseElement, Blake3>::new(&[]);
-    let verifier = FriVerifier::new(&mut channel, &mut coin, options.clone(), max_degree).unwrap();
+    let verifier = FriVerifier::new(&mut channel, &mut coin, options.clone(), max_degree)?;
     let queried_evaluations = positions
         .iter()
         .map(|&p| evaluations[p])
         .collect::<Vec<_>>();
     verifier.verify(&mut channel, &queried_evaluations, &positions)
 }
+
+fn fri_prove_verify_exponent(
```

Comment:
I'd probably call this `fri_prove_verify_helper` or maybe even just `fri_prove_verify` and then rename tests to indicate what we are actually testing - e.g, `fri_folding_2`, `fri_folding_4` etc.